apps/1d-stencil/1d-cuda.cpp

```diff
@@ -5,6 +5,8 @@
 #include <thrust/execution_policy.h>
 #include <thrust/sequence.h>
 
+#include <experimental/mdspan>
```

Comment:
If it is being used in all 1d-xx.cpp files then this should go into a common header file.